frontend/public/components/modals/delete-namespace-modal.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
```

Comment:
@jcaianirh FYI, this will conflict with the i18n work you're doing for the Home nav section.